airflow/models/xcom.py

```diff
@@ -131,6 +131,12 @@ def init_on_load(self):
         self.value = self.orm_deserialize_value()
 
     def __repr__(self):
+        """Return the XCom properties.
```

Comment:
`__repr__` is stand for string representation
`__str__` is also sting representation, however it usual use when you need to convert object to string (every object in Python in theory should be convertable to sting), e.g. `str(object)` or f"Foo Bar {object}"

https://docs.python.org/3/reference/datamodel.html#object.__repr__